guides/intro_to_git.md

```diff
@@ -0,0 +1,316 @@
+
+![](../images/project_structure.png)
+
+* Steps [3](01_intro_to_git.md#3-create-and-switch-to-a-new-branch), [4](01_intro_to_git.md#4-add-a-new-file) and [5](01_intro_to_git.md#5-commit-your-changes-follow-basic-git-command-workflow) will be the same, the only change being that instead of using your local terminal to input Git commands, you will be using the RStudio Cloud built-in terminal.
+
+![](../images/terminal.png)
+
+# General: How to submit a pull request
+
+**Pull request (PR)** is the application you submit on the GitHub repository, to announce to other collaborators working on the repository that you have a new change ready to be merged/part of the main build, the main branch of the repository. Then, either your collaborators or someone you assign will review the change and the branch you have been working on to determine whether the change is ready to be merged with the main branch. The equivalent term for GitLab users is Merge Request (MR).
+
+To submit a pull request:
+
+1. Simply head over to GitHub and to the respective repository
+
+2. Select Pull requests on the top bar menu. 
+
+3. Either select New Pull Request (big green button) on the right hand side or select Compare & pull request if you recently pushed new changes to the branch.
+
+4. Base ref will default to the main branch, head ref will be the branch you wish to merge with the main branch. You can change any of these options, there will be occasions you won't always aim to merge every branch to the main branch.
+
+5. Click Create Pull Request.
+
+6. Fill out any information related to the Pull request, title, description, any comments. On the rigth hand side you can assign yourself as the Assignee, and assign someone in your team to be the Reviewer. Then, select Create Pull Request.
+
+# General: How to accept a pull request
+
+Once you are assigned the pull request's reviewer, you should receive an email notification from GitHub with a link to the pull request's webpage. You can also view any active pull requests by clicking on Pull Requests on the top bar menu.
+
+![](../images/pull_requests.png)
+
+On the Pull request page, there are four tabs: Conversation, Commits, Checks and Files changed:
+
+### Conversation
+
+**Conversation** is the main pull request page. It contains the pull request's description, timeline and comments left by the assignee or reviewer. To merge a pull request you have 3 presented options:
+
+* Merge pull request: this will keep all commits history of the source branch and will be added to the target branch. Creates a symbolic commit.
+* Squash and merge: ticking this will merge the history of commits for the source branch into one commit. This can be useful when the pull request is not a major update and you don't want your repository's history of commits to contain commits with messages such as "fixed typo", "replaced incorrectly placed image". Creates a symbolic commit.
```

Comment:
Not sure 'creates a symbolic commit' is needed here as it's not clear what this is.